core/wiki/macros/sidebar-resizer.tid

```diff
@@ -233,54 +253,72 @@ title: $:/core/procedures/sidebar-resizer
 \end
 
 \function get.drag.diff()
-[<event-fromcatcher-posx>subtract{$:/state/sidebar/resizing!!start-posx}]
+[<event-fromcatcher-posx>subtract<startPosX>]
 :map[<get.theme.option sidebarposition>match[left]then<currentTiddler>multiply[-1]else<currentTiddler>]
 \end
 
 \procedure sidebar-resizer-pointermove-actions()
-<%if [[$:/state/sidebar/resizing]!is[missing]] %>
-	<$let widgetNodeWidth={{$:/state/sidebar/resizing!!widget-node-width}} dragDiff=<<get.drag.diff>> startStoryRight={{{ [convert.to.pixels.locally{$:/state/sidebar/resizing!!start-story-right}] }}} storyRiverWidth={{{ [<startStoryRight>add<dragDiff>] }}} startSidebarWidth={{{ [convert.to.pixels.locally{$:/state/sidebar/resizing!!start-sidebar-width}] }}} sidebarWidth={{{ [<startSidebarWidth>subtract<dragDiff>] }}}>
+<%if [<get.resizer.state>!is[missing]] %>
```

Comment:
would this one work with `[<get.resizer.state>is[tiddler]]` -- So we avoid negation "!"